### PR TITLE
New parameter for information barriers

### PR DESCRIPTION
**Pull request by nibandyo**
Adding new parameter 'InformationSegment' to get-sposite. This parameter provides the collection of segments that are associated with the site.

### CODE REVIEW

File: sharepoint/sharepoint-ps/sharepoint-online/Get-SPOSite.md [hunk 1]
```diff
@@ -101,6 +101,14 @@ Get-SPOSite -Identity https://contoso.sharepoint.com/sites/groupname -detailed |
 
 This example gets quota details for a Group Site.
 
+### -----------------------EXAMPLE 6-----------------------------
+
+```powershell
+Get-SPOSite -Identity [https://contoso.sharepoint.com/sites/research] | Select InformationSegment
```
Comment (get-itips) [inline sharepoint/sharepoint-ps/sharepoint-online/Get-SPOSite.md:107]:
```suggestion
Get-SPOSite -Identity https://contoso.sharepoint.com/sites/research | Select InformationSegment
```

File: sharepoint/sharepoint-ps/sharepoint-online/Get-SPOSite.md [hunk 2]
```diff
@@ -251,6 +259,21 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### InformationSegment
+This parameter displays the segments associated with a site. 
+
+It is applicable for tenants who have enabled M365 Information barriers capability. Please read (https://<kaarin to provide the link ) documentation to understand Information barriers with SharePoint Online.
```
Comment (get-itips) [inline sharepoint/sharepoint-ps/sharepoint-online/Get-SPOSite.md:265]: _Please read_ url link is missing

Comment (dariomws) [inline sharepoint/sharepoint-ps/sharepoint-online/Get-SPOSite.md:265]:
```suggestion
It is applicable for tenants who have enabled Microsoft 365 Information barriers capability. Please read (https://<kaarin to provide the link ) documentation to understand Information barriers with SharePoint Online.
```

**Comment by kenwith** [comment]
@kaarins - see note above.. @nibandyo this will go live so please update the link ASAP.

**Pull request merged**